spec/factories/curator/filestreams/texts.rb

```diff
@@ -7,7 +7,16 @@
     file_set_type { 'Curator::Filestreams::Text' }
     file_name_base { Faker::Books::Lovecraft.tome }
     position { 1 }
-    pagination { {} }
+    hand_side { 'left' }
+    page_type { 'TOC' }
+    page_label { 'My OCR' }
```

Comment:
`hand_side`, `page_type`, and `page_label` would not be used with this FileSet type